### DISCUSSION TITLE
Incremental refresh with CostSummary Power BI FinOps report

### DISCUSSION THREAD

**Discussion by tech2sh** [microsoft/finops-toolkit#942] [General]
Hi,

With latest FinOps hub, the CostSummary Power BI report ( https://microsoft.github.io/finops-toolkit/power-bi/cost-summary)  works fine on Power BI Service. However with Power BI Pro and premium capacity, incremental refresh fails with error "Data source error: Expression.Error: We cannot convert the value null to type Table.. . The exception was raised by the IDbCommand interface.". Anybody faced the similar issue?

**Comment by flanakin** [comment]
@MSBrett have you seen this before?
